<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="PlaceForText">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>291</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="PlaceForAnswer">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>100</y>
      <width>291</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>100</y>
      <width>89</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="main_button">
       <property name="text">
        <string>Find Repetitions</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="second_button">
       <property name="text">
        <string>Find Mistakes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="third_button">
       <property name="text">
        <string>Help to Improve</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>PlaceForAnswer</zorder>
   <zorder>PlaceForText</zorder>
   <zorder>verticalLayoutWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
